### ISSUE TITLE
How to understand the instruction in the step 3 when i have only 2 gpus?

### ISSUE THREAD

**Issue by RockJim2001** [fanq15/FewX#77]
At first, i'm confused the content that is "Run sh all.sh in the root dir. (This script uses 4 GPUs. You can change the GPU number. If you use 2 GPUs with unchanged batch size (8), please https://github.com/fanq15/FewX/issues/6#issuecomment-674367388.)".   The training technology called “Distributed Data Parallel(DDP)”  is used to train the model with 4 gpus, 8 batch size,0.004 base_lr. When i want to train the model with 2gpu, the batch size is unchanged which is equality to the batch size of per gus is double and need't to halve the learning rate.

With this doubt， I train the model with 2 gpus, 8 batch size, half of the learning rate during the whole processing(train and finetune). The result i got is the following:
1)、 train:
`[04/05 23:17:33] fewx.evaluation.coco_evaluation INFO: Evaluation results for bbox: 
|  AP   |  AP50  |  AP75  |  APs  |  APm  |  APl  |
|:-----:|:------:|:------:|:-----:|:-----:|:-----:|
| 1.083 | 2.252  | 0.951  | 0.904 | 1.366 | 1.640 |
[04/05 23:17:33] fewx.evaluation.coco_evaluation INFO: Evaluation results for VOC 20 categories =======> AP  : 4.33
[04/05 23:17:33] fewx.evaluation.coco_evaluation INFO: Evaluation results for VOC 20 categories =======> AP50: 9.01
[04/05 23:17:33] fewx.evaluation.coco_evaluation INFO: Evaluation results for VOC 20 categories =======> AP75: 3.80
[04/05 23:17:33] fewx.evaluation.coco_evaluation INFO: Evaluation results for VOC 20 categories =======> APs : 3.61
[04/05 23:17:33] fewx.evaluation.coco_evaluation INFO: Evaluation results for VOC 20 categories =======> APm : 5.46
[04/05 23:17:33] fewx.evaluation.coco_evaluation INFO: Evaluation results for VOC 20 categories =======> APl : 6.56
[04/05 23:17:33] fewx.evaluation.coco_evaluation INFO: Evaluation results for Non VOC 60 categories =======> AP  : 0.00
[04/05 23:17:33] fewx.evaluation.coco_evaluation INFO: Evaluation results for Non VOC 60 categories =======> AP50: 0.00
[04/05 23:17:33] fewx.evaluation.coco_evaluation INFO: Evaluation results for Non VOC 60 categories =======> AP75: 0.00
[04/05 23:17:33] fewx.evaluation.coco_evaluation INFO: Evaluation results for Non VOC 60 categories =======> APs : 0.00
[04/05 23:17:33] fewx.evaluation.coco_evaluation INFO: Evaluation results for Non VOC 60 categories =======> APm : 0.00
[04/05 23:17:33] fewx.evaluation.coco_evaluation INFO: Evaluation results for Non VOC 60 categories =======> APl : 0.00`
2)、finetune:
`[04/06 14:35:51] fewx.evaluation.coco_evaluation INFO: Evaluation results for bbox: 
|  AP   |  AP50  |  AP75  |  APs  |  APm  |  APl  |
|:-----:|:------:|:------:|:-----:|:-----:|:-----:|
| 0.720 | 1.649  | 0.542  | 0.428 | 0.916 | 1.055 |
[04/06 14:35:51] fewx.evaluation.coco_evaluation INFO: Evaluation results for VOC 20 categories =======> AP  : 2.88
[04/06 14:35:51] fewx.evaluation.coco_evaluation INFO: Evaluation results for VOC 20 categories =======> AP50: 6.60
[04/06 14:35:51] fewx.evaluation.coco_evaluation INFO: Evaluation results for VOC 20 categories =======> AP75: 2.17
[04/06 14:35:51] fewx.evaluation.coco_evaluation INFO: Evaluation results for VOC 20 categories =======> APs : 1.71
[04/06 14:35:51] fewx.evaluation.coco_evaluation INFO: Evaluation results for VOC 20 categories =======> APm : 3.66
[04/06 14:35:51] fewx.evaluation.coco_evaluation INFO: Evaluation results for VOC 20 categories =======> APl : 4.22
[04/06 14:35:51] fewx.evaluation.coco_evaluation INFO: Evaluation results for Non VOC 60 categories =======> AP  : 0.00
[04/06 14:35:51] fewx.evaluation.coco_evaluation INFO: Evaluation results for Non VOC 60 categories =======> AP50: 0.00
[04/06 14:35:51] fewx.evaluation.coco_evaluation INFO: Evaluation results for Non VOC 60 categories =======> AP75: 0.00
[04/06 14:35:51] fewx.evaluation.coco_evaluation INFO: Evaluation results for Non VOC 60 categories =======> APs : 0.00
[04/06 14:35:51] fewx.evaluation.coco_evaluation INFO: Evaluation results for Non VOC 60 categories =======> APm : 0.00
[04/06 14:35:51] fewx.evaluation.coco_evaluation INFO: Evaluation results for Non VOC 60 categories =======> APl : 0.00`
3)、 test:
`[04/06 15:20:07] fewx.evaluation.coco_evaluation INFO: Evaluation results for bbox: 
|  AP   |  AP50  |  AP75  |  APs  |  APm  |  APl  |
|:-----:|:------:|:------:|:-----:|:-----:|:-----:|
| 0.720 | 1.649  | 0.542  | 0.428 | 0.916 | 1.055 |
[04/06 15:20:07] fewx.evaluation.coco_evaluation INFO: Evaluation results for VOC 20 categories =======> AP  : 2.88
[04/06 15:20:07] fewx.evaluation.coco_evaluation INFO: Evaluation results for VOC 20 categories =======> AP50: 6.60
[04/06 15:20:07] fewx.evaluation.coco_evaluation INFO: Evaluation results for VOC 20 categories =======> AP75: 2.17
[04/06 15:20:07] fewx.evaluation.coco_evaluation INFO: Evaluation results for VOC 20 categories =======> APs : 1.71
[04/06 15:20:07] fewx.evaluation.coco_evaluation INFO: Evaluation results for VOC 20 categories =======> APm : 3.66
[04/06 15:20:07] fewx.evaluation.coco_evaluation INFO: Evaluation results for VOC 20 categories =======> APl : 4.22
[04/06 15:20:07] fewx.evaluation.coco_evaluation INFO: Evaluation results for Non VOC 60 categories =======> AP  : 0.00
[04/06 15:20:07] fewx.evaluation.coco_evaluation INFO: Evaluation results for Non VOC 60 categories =======> AP50: 0.00
[04/06 15:20:07] fewx.evaluation.coco_evaluation INFO: Evaluation results for Non VOC 60 categories =======> AP75: 0.00
[04/06 15:20:07] fewx.evaluation.coco_evaluation INFO: Evaluation results for Non VOC 60 categories =======> APs : 0.00
[04/06 15:20:07] fewx.evaluation.coco_evaluation INFO: Evaluation results for Non VOC 60 categories =======> APm : 0.00
[04/06 15:20:07] fewx.evaluation.coco_evaluation INFO: Evaluation results for Non VOC 60 categories =======> APl : 0.00`

The result has a large distance compared with the result of the code.And then i finetune the model with the pretrain weight of the base   
 model provided this code.However, i still not get the ideal result.

I would be grateful if anyone, especially the author, could give guidance！ @fanq15

**Comment by RockJim2001** [comment]
Sorry that there is a problem with my formatting that affects your reading.Please forgive me for being a newbie！
